.github/workflows/main.yaml

```diff
@@ -5,9 +5,7 @@ on:
     branches:
       - main
   push:
-    branches:
-      - main
-    schedule:
+  schedule:
```

Comment:
Note that the schedule was indented correctly. It is possible that this job has not been running daily as expected.